.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "src/core_ocean/cvmix"]
+	path = src/core_ocean/cvmix
+	url = git@github.com:CVMix/CVMix-src.git
+[submodule "src/core_ocean/BGC"]
+	path = src/core_ocean/BGC
+	url = git@github.com:E3SM-Project/Ocean-BGC.git
```

Comment:
@mgduda @matthewhoffman @akturner, we would like to add two submodules for use by the ocean core. This file needs to be added to the top level MPAS directory. It will not affect any other cores, but I wanted to ask your approval. If you want to add submodules to your core at a later time, you would add them to this file. On a new clone or branch, use
```
git submodule update --init
```
and it will fill in the submodules for ALL cores, not just yours. But I think that behavior is fine. That is what we do in the E3SM repo.

Reply:
Hmm, I hadn't thought of that.  This could become irritating if the submodules become large but I guess we'll cross that bridge when we come to it.

Cores other that the ocean should still work fine without `git submodule update --init`, right?

Reply:
Also, maybe this means these changes need to go to `develop`, not `ocean/develop`.

Reply:
Since this is a new file and an essential part of this PR, it's OK to merge into `ocean/develop`. I think having submodules for many cores (potentially) in the future is not problematic. The user can either get them all with one command:
```
git submodule update --init
```
or just get the ones relevant to their own core, i.e.
```
git submodule update --init src/core_ocean/BGC
git submodule update --init src/core_ocean/cvmix
```
Either way is not much trouble.

Reply:
@mgduda do you approve the addition of this `.gitmodules` file?

Reply:
The addition of a .gitmodules file in the root directory makes sense to me.